script.js

```diff
@@ -42,9 +42,143 @@ contactLink.addEventListener('click', (e) => {
+      </div>
+      <div class="popup-footer">
+        <p class="work-descripton-mobile">${this.summary}</p>
+        <p class="work-description-desktop">${this.summary}
+         typesetting industry. Lorem Ipsum
+         has been the industry's
+          standard dummy text ever 
+          since the 1500s, when an unknown printer took a
+           galley of type and scrambled it 1960s with the
+          releorem Ipsum is simply dummy text of the printing and
+           typesetting industry. Lorem Ipsum han printer took a galley of
+          type and scrambled it 1960s with the releawn printer took 
+          a galley of type and scrambled it 1960s with the releaLorem
+          Ipsum is simply dummy text of the printing and typesetting
+           industry. Lorem Ipsum has been the industry's standard dummy
+          text ever since the 1500s, when an unknown printer took a galley of
+           type and scrambled it 1960s with the relea
+        </p>
+        <div class="popup-tools-and-btn">
+          <div class="popup-work__tools">${this.usedTools}
+          </div>
+          <div class="popup-work__tools popup-work-tools__desktop">
+            <div class="tool">Github</div>
+            <div class="tool">Ruby</div>
+            <div class="tool">Bootstrap</div>
+          </div>
+          <div class="popup-btn__container">
+            <button class="popup-btn portfolio-btn">
+              See live <img src="icons/popup-see-project.png" alt="icon"
+                class="popup-btn__icon"></button>
+            <button class="popup-btn portfolio-btn">
+              See Source <img src="icons/popup-github.png" alt="icon"
+                class="popup-btn__icon"></button>
+          </div>
+        </div>
+      </div>
+    </div>`;
+    popup.classList.remove('popup-hidden');
+    popup.querySelector('.close-popup__btn').addEventListener('click', ()=> {
+      popup.classList.add('popup-hidden');
+    });
+    popup.addEventListener('click', (e) =>{
+      if (e.target.classList.contains('popup')) {
+        popup.classList.add('popup-hidden');
+      }
+    });
+  }
+
+  /**
+ * this function shows programaticaly the work cards on the page
+ */
+
+  showWorkCard() {
+    workSection.insertAdjacentHTML('beforeend', `
+    <div class="work" id="${this.id}">
+      <img src="${this.imgSrc}" class="work-image" alt="work image">
+      <div class="work-description">
+        <h3 class="work-name title">${this.name}</h3>
+        <h4 class="small-description">
+        ${this.smallDescription}
+        </h4>
+        <p class="work-summary">${this.summary}
+        </p>
+        <ul class="work-used-tools">${this.usedTools}
+        </ul>
+          <button class="portfolio-btn" id="see-project-btn-${this.id}">
+            See Project</button>
+      </div>
+    </div>
+    `);
+    workSection.querySelector(`#see-project-btn-${this.id}`)
+        .addEventListener('click', this.displayPopup.bind(this));
+  }
+}
```

Comment:
- [ ] While you did a good job creating the work section :+1:, I think it is one of the [main requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) to create dynamically the original cards that you previously created in HTML by using javascript so kindly remove all the code for all the cards from the HTML (the projects/works section) and create it by using only JS. It would help you to learn how to create HTML by JS :100:. [Here](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) you can find a good example of it.